src/puppetlabs/puppetdb/http/edges.clj

```diff
@@ -0,0 +1,36 @@
+(ns puppetlabs.puppetdb.http.edges
+  (:require [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.http :as http]
+            [puppetlabs.puppetdb.http.query :as http-q]
+            [puppetlabs.puppetdb.jdbc :refer [with-transacted-connection]]
+            [puppetlabs.puppetdb.middleware :refer [verify-accepts-json validate-query-params
+                                                    wrap-with-paging-options]]
+            [puppetlabs.puppetdb.query-eng :refer [produce-streaming-body]]
+            [puppetlabs.puppetdb.query.paging :as paging]))
+
+(defn build-edges-app
+  [version entity]
+  (comp
+   (fn [{:keys [params globals paging-options]}]
+     (produce-streaming-body
+      entity
+      version
+      (params "query")
+      paging-options
+      (:scf-read-db globals)
+      (:url-prefix globals)))
+   http-q/restrict-query-to-active-nodes))
```

Comment:
we don't restrict on the catalogs app, so I think deactivated nodes will produce misleading hrefs for hsqldb users